app/policies/project_test_policy.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2015 ProbeDock
+# Copyright (c) 2012-2014 Lotaris SA
+#
+# This file is part of ProbeDock.
+#
+# ProbeDock is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ProbeDock is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ProbeDock.  If not, see <http://www.gnu.org/licenses/>.
+class ProjectTestPolicy < ApplicationPolicy
+  def show?
+    user.try(:is?, :admin) || record.project.organization.try(:public?) || user.try('member_of?', record.project.try(:organization))
```

Comment:
Good. I know the existing policies don't all use the helpers because I haven't yet taken the time to refactor all of them.